packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -186,4 +204,87 @@ void main() {
       greaterThan(tester.getBottomLeft(find.byKey(avatarA)).dy)
     );
   });
+
+  testWidgets('UserAccountsDrawerHeader provides semantics', (WidgetTester tester) async {
+    final SemanticsTester semantics = new SemanticsTester(tester);
+    await pumpTestWidget(tester);
+    expect(
+      semantics,
+      hasSemantics(
+        new TestSemantics(
+          children: <TestSemantics>[
+            new TestSemantics(
+              label: 'Signed in\nA\nname\nemail',
+              textDirection: TextDirection.ltr,
+              children: <TestSemantics>[
+                new TestSemantics(
+                  label: r'B',
+                  textDirection: TextDirection.ltr,
+                ),
+                new TestSemantics(
+                  label: r'C',
+                  textDirection: TextDirection.ltr,
+                ),
+                new TestSemantics(
+                  label: r'D',
+                  textDirection: TextDirection.ltr,
+                ),
+                new TestSemantics(
```

Comment:
This empty SemanticsNode seems strange. Its child should just be a child of its parent, I think.

Reply:
Removed.